tests/core/pyspec/eth2spec/test/bellatrix/confirmation_rule/test_confirmation_rule.py

```diff
@@ -0,0 +1,671 @@
+from eth2spec.test.context import spec_state_test, with_config_overrides, with_presets, with_bellatrix_and_later
+from eth2spec.test.helpers.attestations import (
+    get_valid_attestation_at_slot,
+    next_epoch_with_attestations,
+    next_slots_with_attestations,
+)
+from eth2spec.test.helpers.constants import MINIMAL
+from eth2spec.test.helpers.fork_choice import (
+    add_block,
+    get_genesis_forkchoice_store_and_block,
+)
+
+from eth2spec.test.helpers.state import (
+    next_epoch,
+)
+
+
+def on_tick_step(spec, store, time, test_steps):
+    spec.on_tick(store, time)
+    test_steps.append({"tick": int(time)})
```

Comment:
Do you intentionally want to avoid calling `output_store_checks` because the test format is not shared? If so, I agree, as @mkalinin suggested, it would be nice to apply the optimistic sync style test format - inheriting the common `checks` from the FC test format.

Reply:
Yep, I intentionally avoid calling `output_store_checks`.
However, this is not because the test format is not shared.
It is because in these tests, we are only interested in checking the confirmation rule behaviour. We are not interested  in the checks outputted by output_store_checks`